src/Coin.cpp

```diff
@@ -297,6 +305,15 @@ DerivationPath TW::derivationPath(TWCoinType coin) {
             DerivationPathIndex(0, true),
         };
 
+    case TWCoinTypeCosmos:
+        return DerivationPath{
+            DerivationPathIndex(purpose(coin), true),
+            DerivationPathIndex(coin, true),
```

Comment:
@johnnynanjiang did you verify this with ledger? 

I would expect it should use the same format as binance or ethereum

Reply:
Yes, @johnnynanjiang did